app/components/works/version_description_component.html.erb

```diff
@@ -1,47 +1,52 @@
+          </div>
+        </div>
+        <div class='mb-3 row'>
+          <div class='col-sm-10 form-check'>
+            <%= form.radio_button :new_user_version, 'no', required: true, data: { action: 'click->new-user-version#displayVersionDescription', new_user_version_target: 'userVersionNo' }, class: 'form-check-input' %>
+            <%= form.label :new_user_version, 'No', value: 'no', class: 'form-check-label fw-semibold' %>
+            <ul>
+              <li>Edit form fields only. Files may not be changed.</li>
+              <li>The version number of this deposit will not change.</li>
+              <li>Changes you make to this item will appear as updates to the current version on the PURL page.</li>
+            </ul>
+            <%= form.label :version_description, "What's changing?", class: 'col-form-label' %>
+            <%= form.text_field :version_description, data: { new_user_version_target: 'versionDescriptionNo' }, class: 'form-control' %>
+          </div>
         </div>
       </div>
+    </section>
+  <% else %>
+    <section class='version'>
+      <header>Version your work *</header>
       <div class='mb-3 row'>
-        <div class='col-sm-10 form-check'>
-          <%= form.radio_button :new_user_version, 'no', data: { action: 'click->new-user-version#displayVersionDescription', new_user_version_target: 'userVersionNo' }, class: 'form-check-input' %>
-          <%= form.label :new_user_version, 'No', value: 'no', class: 'form-check-label fw-semibold' %>
-          <ul>
-            <li>Edit form fields only. Files may not be changed.</li>
-            <li>The version number of this deposit will not change.</li>
-            <li>Changes you make to this item will appear as updates to the current version on the PURL page.</li>
-          </ul>
-          <%= form.label :current_version_description, "What's changing?", class: 'col-form-label' %>
-          <%= form.text_field :current_version_description, data: { new_user_version_target: 'versionDescriptionNo' }, class: 'form-control' %>
+        <%= form.label :version_description, "What's changing?", class: 'col-sm-2 col-form-label' %>
+        <div class='col-sm-10'>
+          <%= form.text_field :version_description, class: 'form-control', required: true %>
+          <div class='invalid-feedback'>You must describe your changes.</div>
         </div>
       </div>
-    </div>
-  </section>
-<% else %>
-  <section class='version'>
-    <header>Version your work *</header>
-    <div class='mb-3 row'>
-      <%= form.label :version_description, "What's changing?", class: 'col-sm-2 col-form-label' %>
-      <div class='col-sm-10'>
-        <%= form.text_field :version_description, class: 'form-control', required: true %>
-        <div class='invalid-feedback'>You must describe your changes.</div>
-      </div>
-    </div>
-  </section>
+    </section>
+  <% end %>
+<% end %>
+<% if show_hidden? %>
+  <%= form.hidden_field :new_user_version, value: 'no' %>
```

Comment:
There will always be a `new_user_version` param (either selected by the user or hidden).